### DISCUSSION TITLE
Extension cable for CTs?

### DISCUSSION THREAD

**Discussion by pyrodex** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#143] [Q&A]
I used to use AccuCT's with my IoTaWatt setup and switched/swapped them all out for the YHDC CTs and so far so good except for one issue. There are a few CTs where the cable isn't going to make it all the way back to my ESP unit and was wondering if I could use a standard 3.5MM extension cable or would that increase resistance or cause any accuracy issues? 

For example something like this:

https://www.amazon.com/CableCreation-Female-Extension-Adapter-Connector/dp/B01K3WYDIE

I was thinking no more than 3'-10' max to give me some better final state setup and a clean design. 

Thoughts before I go invest in a few cables if this will work or has been tested?

Thanks community!

**Comment by CircuitSetup** [comment]
Yes, you can use those. I would recommend calibrating the CTs with extensions plugged in since the length could introduce some additional resistance.

**Comment by spikeygg** [comment]
I did a bit of research on this subject and found this page: https://ctlsys.com/support/current-transformer-ct-wire-extension/ which says you'll likely want twisted pair for noise immunity. I'm thinking I'll use some CAT5e ethernet cable to extend all my CT sensor wires. As CircuitSetup said, a recalibration would be needed if you were using them before extending the wires.
